_test/test_utilities/test_split_vector_max_sum.m

```diff
@@ -35,39 +35,39 @@ function test_outputs_have_length_1_if_max_counts_eq_total_counts(~)
 
     function test_error_if_counts_is_not_a_vector(~)
         vector = ones(2, 10);
-        f = @() split_vector(vector, 100);
+        f = @() split_vector_max_sum(vector, 100);
         assertExceptionThrown(f, 'MATLAB:expectedVector');
     end
 
     function test_error_if_any_value_of_counts_is_negative(~)
         vector = ones(1, 10);
         vector(5) = -1;
-        f = @() split_vector(vector, 100);
+        f = @() split_vector_max_sum(vector, 100);
         assertExceptionThrown(f, 'MATLAB:expectedNonnegative');
     end
 
     function test_error_if_max_counts_is_not_a_scalar(~)
         vector = ones(1, 10);
-        f = @() split_vector(vector, [1, 1]);
+        f = @() split_vector_max_sum(vector, [1, 1]);
         assertExceptionThrown(f, 'MATLAB:expectedScalar');
     end
 
     function test_error_if_max_counts_is_zero(~)
         vector = ones(1, 10);
-        f = @() split_vector(vector, 0);
+        f = @() split_vector_max_sum(vector, 0);
         assertExceptionThrown(f, 'MATLAB:expectedPositive');
     end
 
     function test_error_if_max_counts_is_negative(~)
         vector = ones(1, 10);
-        f = @() split_vector(vector, -1);
+        f = @() split_vector_max_sum(vector, -1);
         assertExceptionThrown(f, 'MATLAB:expectedPositive');
     end
 
     function test_chunking_correct_if_a_count_gt_max_counts(~)
```

Comment:
"chunking correct if count gt max counts" doesn't tell me what "correct is": does it give me a chunk with more than max elements, or multiple chunks with max elements?

Reply:
Is `test_vector_value_gt_max_chunk_sum_comprises_its_own_chunk` a better name? It describes the test's purpose, but maybe doesn't describe that it's actually verifying that the rest of the chunking is "correct" as well.